### DISCUSSION TITLE
aiomysql executemany is very slow to insert

### DISCUSSION THREAD

**Discussion by RuoCJ** [aio-libs/aiomysql#983] [Q&A] [answered]
I used aiomysql executemany to insert over 800 pieces of data, which took over thirty seconds, but pymysql only took over two seconds。

**Comment by dodysw2** [comment, marked as answer]
Hint, check your insert query if it matches this regex at this line: https://github.com/aio-libs/aiomysql/blob/83aa96e12b1b3f2bd373f60a9c585b6e73f40f52/aiomysql/cursors.py#L19

There's a logic that if the query doesn't match, it will turn executemany into individual insert queries instead of a batched one.
